src/main/webapp/app/tasks/local-task.service.ts

```diff
@@ -40,4 +41,22 @@ export class LocalTaskService implements TaskService {
   private writeTasks(tasks: Task[]): void {
     localStorage.setItem(LocalTaskService.STORAGE_KEY, JSON.stringify(tasks));
   }
+  
+  update(id: string, checked:boolean): Observable<void> {
+    const tasks = this.readTasks();
+    const index = tasks.findIndex(task => task.id === id);
+    if (index !== -1) {
+      let tempTask = tasks[index];
+      tempTask.checked = checked;
+      tasks.splice(index, 1);
+      tasks.push(tempTask);
+      this.writeTasks(tasks);
+    }
+    return of(null);
+  }
+
+  deleteDone(): Observable<void>{
+    this.writeTasks(this.readTasks().filter(x=> x.checked == false));
```

Comment:
Missing whitespace and == should be ===. This is one of the many places of lint errors. You would have easily found them by running `yarn lint`.